.github/workflows/npc_db_validation.yml

```
@@ -0,0 +1,91 @@
+name: Validate NPC Scripts and DB Changes
+# npc_db_validation.yml
+
+# For NPC and DB validation we only need two builds: one of Renewal and one for Pre-Renewal checks.
+# NPC scripts and database files are not platform dependent, so we can achieve this validation using only a simple linux setup.
+
+on: [push, pull_request]
+  paths:
+    # This workflow should run when a file in either the db/ or npc/ directory has been modified.
+    - 'db/**'
+    - 'npc/**'
+
+jobs:
+  build:
+    # Github Actions checks for 'ci skip', 'skip ci', but not a hyphenated version.
```

```suggestion
    # Github Actions checks for '[ci skip]', '[skip ci]', '[no ci]', '[skip actions]', or '[actions skip]' but not a hyphenated version.
```
Just to sync with the other PR.